interface/billing/billing_report.php

```diff
@@ -1463,7 +1463,7 @@ function divtoggle(spanid, divid) { //Called in the Expand, Collapse links(This
             $('#update-tooltip').attr("title", <?php echo xlj('Click Update List to display billing information filtered by the selected Current Criteria'); ?>).tooltip();
         });
     </script>
-    <input type="hidden" name="divnos" id="divnos" value="<?php echo attr($divnos) ?>" />
+    <input type="hidden" name="divnos" id="divnos" value="<?php echo attr($divnos ?? null) ?>" />
```

Comment:
when putting something through a basic text manipulation function, better to convert to '' rather than null:
```
$divnos ?? ''

```
(yep, it all hurts the brain; php 8.1 is throwing warnings when sending null to its string manipulation functions)